bench/workgen/workgen.cxx

```diff
@@ -2324,6 +2583,24 @@ WorkloadOptions::WorkloadOptions()
       "timestamp forward");
     _options.add_bool("max_idle_table_cycle_fatal", max_idle_table_cycle_fatal,
       "print warning (false) or abort (true) of max_idle_table_cycle failure");
+    _options.add_int("dynamic_create_period", dynamic_create_period,
```

Comment:
It looks like `interval` is used rather than `period` in other workload options, maybe change to `create_interval` and `drop_interval`?

Reply:
Done.